lib/Auth/utils.php

```diff
@@ -86,6 +98,20 @@ public function fetchAuthToken($serverconfig) {
 		return $token;
 	}
 
+	/**
+	 * This function can edit the state to enter the needed token types for a user.
+	 * If the user needs u2f, the u2f sign request will be triggered.
```

Comment:
This does not happen in this function, so I would not mention it here.

This function simply adds new entries to the $state object, depending on the $body. Right?
So simply mention, which keys are added in the $state.

Reply:
fixed in dae2c98